### PR TITLE
New package: xkblayout-state-1

### PR DESCRIPTION
**Pull request by DirectorX**

### CODE REVIEW

**Comment by Duncaen** [comment]
Please force push, instead of closing and opening a new PR next time.

**Comment by DirectorX** [comment]
@Duncaen, i tried many times...

**Comment by Vaelatern** [comment]
@DirectorX The workflow that would have likely fixed that problem: `git rebase master; git push -f`

File: srcpkgs/xkblayout-state/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+pkgname=xkblayout-state
+version=1
+revision=1
+license="GPL-2"
+homepage="https://github.com/nonpop/xkblayout-state"
+makedepends="libX11-devel"
+build_style=gnu-makefile
+short_desc="A small command-line program to get/set the current keyboard layout"
+maintainer="DirectorX <DirectorX@users.noreply.github.com>"
+distfiles="https://github.com/nonpop/${pkgname}/archive/v${version}.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/xkblayout-state/template]: Can we use `${homepage}`?

**Comment by Vaelatern** [comment]
Can you try and merge your makefile patch upstream? Not necessary before the package is merged, but will greatly reduce the burden of maintaining a patch.

File: srcpkgs/xkblayout-state/template (hunk 1 above)
Comment (DirectorX) [inline srcpkgs/xkblayout-state/template]: sure thing

**Comment by DirectorX** [comment]
@Vaelatern i will try merge my makefile patch upstream.. from my point of view there is no point in doing that. `xkblayout-state` project is not maintained consistently.

**Pull request merged**